### PR TITLE
EVSS client tests: Add VCR for endpoint mocking

### PR DESCRIPTION
**Pull request by earthboundkid**
I don't know if the standard is to check in VCR cassettes or gitignore them. I have them here for now.

### CODE REVIEW

**Comment by saneshark** [comment]
standard is to check them in, and only delete them when you need to hit external endpoints again for fresh data / tests. That way Travis never has to make external requests.

File: spec/client_spec.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
```
Comment (austin-martinez) [inline spec/client_spec.rb]: can we move this file to `/spec/lib/evss_spec.rb`?

**Comment by austin-martinez** [comment]
Can you make sure rubocop runs without any offenses? I have been ignoring the single quote vs. double quote rule so far but all other rules should pass in our PRs

**Comment by earthboundkid** [comment]
I hadn't been using rubocop. I'll get it set up locally.

**Comment by earthboundkid** [comment]
It should have all of the issues Austin mentioned fixed now.

**Comment by austin-martinez** [comment]
LGTM 🍶

**Pull request merged**